### PR TITLE
Trivial fixes to cleanup and improve readability

### PR DESCRIPTION
**Pull request by kpouget**

### CODE REVIEW

**Comment by ArangoGutierrez** [comment]
Thanks for this @kpouget !!! really appreciated! I will take a look at this PR, so  far looks good, just need to test it

**Review by ArangoGutierrez — APPROVED** [review]
/lgtm

**Comment by kpouget** [comment]
@ArangoGutierrez can you (re) LGTM this one ?

File: build/root/usr/local/bin/run [hunk 1]
```diff
@@ -13,10 +13,10 @@ set -o nounset
 
 if [[ -z "${KUBECONFIG}" ]]
 then
-		echo "No kubeconfig, can't continue"
-		exit 1
+    echo "No kubeconfig, can't continue"
+    exit 1
 fi
-echo "Kubeconfiug found at ${KUBECONFIG}, proceeding with tests"
+echo "Kubeconfug found at ${KUBECONFIG}, proceeding with tests"
```
Comment (ArangoGutierrez) [inline build/root/usr/local/bin/run:19]:
```suggestion
echo "Kubeconfig found at ${KUBECONFIG}, proceeding with tests"
```

**Review by ArangoGutierrez — COMMENTED** [review]
just 1 typo check

**Review by ArangoGutierrez — APPROVED** [review]
/lgtm

**Comment by kpouget** [comment]
oh indeed, I removed the wrong letter in  https://github.com/openshift-psap/ci-artifacts/pull/24/commits/40d0d7899a04328638c36f0d692a5dad54f69601 -,-

**Pull request merged**